src/entities/SaveHistory.ts

```diff
@@ -0,0 +1,83 @@
+import * as env from "@utils/config.constant";
+import fs from 'fs';
+
+interface HistoryEntry {
+  Epoch: number;
+  
+}
+
+export class SaveHistory {
+  private readonly HISTORY_JSON_PATH = env.HISTORY_JSON_PATH;
+  private readonly DISPLAY_LENGTH = env.DISPLAY_LENGTH;
+  private readonly MAX_LENGTH = env.MAX_LENGTH;
+  private readonly CHAIN_CONTROL_HISTORY = env.CHAIN_CONTROL_HISTORY;
+  path = require("path");
```

Comment:
`require("path")` is a commonjs module convention, while `import path from "path"` is used in ESM (which we are using in this project, so please use the latter and put the import at the top.